pkg/cmd/auth.go

```diff
@@ -60,6 +60,7 @@ func (cmd *AuthCommand) createCobraCommand(rootCmd spi.GalasaCommand) (*cobra.Co
 			"enabling secure interactions with the ecosystem.",
 	}
 
+	authCmd.PersistentFlags().AddFlagSet(commsCmd.CobraCommand().PersistentFlags())
```

Comment:
Adding the CommsCommand as a hidden command in the existing command hierarchy (e.g. `root` -> `comms` -> `runs`) caused subcommands to also be hidden.

Since we're only using the comms command to pass down common flags, I've added the comms command's persistent flag set to other commands where needed.

Reply:
Changed CommsCommand into a flagset since we're only interested in passing down flags - see https://github.com/galasa-dev/cli/pull/323/commits/a863c6facc1ef343362bbc600d3b68203a9dad4b